src/components/Details/VmDetails/VmDetails.js

```diff
@@ -312,6 +316,22 @@ export class VmDetails extends React.Component {
                     template={this.state.template}
                   />
                 </dd>
+                <dt>Services</dt>
+                <dd>
+                  {services ? (
+                    services
+                      .filter(service => get(service, ['spec', 'selector', TEMPLATE_VM_NAME_LABEL]) === getName(vm))
```

Comment:
this should go probably to `selectors/service/`

Reply:
moved to `selectors`